src/client/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AndroidDeviceAuthManager.cs

```diff
@@ -4,17 +4,16 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Threading.Tasks;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
 
 namespace Microsoft.Identity.Client.Platforms.Android.EmbeddedWebview
 {
     [global::Android.Runtime.Preserve(AllMembers = true)]
-    internal class DeviceAuthHelper
+    internal class AndroidDeviceAuthManager : IDeviceAuthManager
     {
-        public static bool CanHandleDeviceAuthChallenge {
-            get { return false; }
-        }
+        public bool CanHandleDeviceAuthChallenge { get { return false; } }
 
-        public static Task<string> CreateDeviceAuthChallengeResponseAsync(IDictionary<string, string> challengeData)
+        public Task<string> CreateDeviceAuthChallengeResponseAsync(IDictionary<string, string> challengeData)
```

Comment:
Can be made to do more, e.g. `Task DecorateRequestWithDeviceCodeChallengeAsync(HttpRequestMessage)`.